client/packages/common/src/ui/layout/tables/material-react-table/useSimpleMaterialTable.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { MRT_RowData, MRT_ShowHideColumnsButton } from 'material-react-table';
+import { BaseTableConfig, useBaseMaterialTable } from './useBaseMaterialTable';
+import { Box } from '@mui/material';
+
+interface SimpleTableConfig<T extends MRT_RowData> extends BaseTableConfig<T> {
+  bottomToolbarContent?: string | React.JSX.Element | React.JSX.Element[];
+}
+
+export const useSimpleMaterialTable = <T extends MRT_RowData>({
+  bottomToolbarContent,
+  ...tableOptions
+}: SimpleTableConfig<T>) => {
+  const {
+    getIsPlaceholderRow = () => false,
+    getIsRestrictedRow = () => false,
+  } = tableOptions;
+
+  const table = useBaseMaterialTable<T>({
+    enableRowSelection: false,
+    enableBottomToolbar: !!bottomToolbarContent,
+    enableTopToolbar: false,
+    enableColumnActions: false,
+    enableSorting: false,
+    enableColumnResizing: false,
+    enableColumnPinning: false,
+    enableColumnOrdering: false,
+    state: {
+      ...tableOptions.state,
+      density: 'compact',
```

Comment:
agree need to review the padding/margin on inputs, so the rows actually render densely. Should be fine to expect that other calling components add any necessary margins